ENH: Add habibi.pdf from https://github.com/py-pdf/PyPDF2/issues/1111

dkg opened the pull request.

dkg committed:
ENH: Add habibi.pdf from https://github.com/py-pdf/PyPDF2/issues/1111

MartinThoma commented:
Nice!

Could you please add an entry for the pdf in https://github.com/py-pdf/sample-files/blob/main/files.json? This json allows us to run automated tests over all files (the thought was that other projects could find interesting test cases with that file more easily)

dkg committed:
ENH: add sample pdf with alternate CMap structure

dkg commented:
good call, @MartinThoma !  i'll add both files to `files.json` in a minute here.

dkg committed:
ENH: added both variants of `habibi.pdf` to `files.json`

(note: habibi-oneline-cmap.pdf was made with a manual tweak to
habibi.pdf, as documented in README.md, so i've added "+manual" to the
producer string)

MartinThoma commented:
Thank you :heart:
